Assignment2/toasterdoom-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,50 @@
+{
+    "@context": {
+        "student": "http://fi.upm.es/alumnos/b.agirre",
+        "subject": "http://fi.upm.es/GII/asignaturas"
+    },
+    "student:@id": "b.agirre",
+    "student:@type": "Student",
```

Comment:
"@id" and "@type" are JSON-LD tags, thus remove the prefix. 
You have to add the prefix at the id and type values. e.g. "student:bagirre"